app/views/projects/index.html.slim

```diff
@@ -28,6 +29,10 @@ table.table.table-striped.table-bordered.table-condensed.hidden-sm.hidden-xs
               = icon('minus')
       td = project_tags project
       td
+        - if project.teams.any?
+          - project.teams.each do |team|
+            = link_to team.name, activities_url(team_id: team.id)
```

Comment:
I was about to write "let's add `Team#display_name` here" since that adds the season. But it also adds the project name. Do we need the season name?